lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EnsembleCalibration.py

```diff
@@ -376,6 +376,8 @@ def test_alternative_calibration_name(self):
             self.historic_temperature_forecast_cube,
             self.temperature_truth_cube)
         self.assertIsInstance(result, tuple)
+        self.assertIsInstance(result[0], iris.cube.Cube)
```

Comment:
The doc-string for the test refers to a cubelist, where as you get cubes back (they just happen to be in a tuple right?).

Reply:
I've amended what is returned for this unit test.